llvm/test/Transforms/LoopVectorize/ARM/mve-known-trip-count.ll

```diff
@@ -387,10 +387,10 @@ for.body:                                         ; preds = %entry, %for.body
 }
 
 ; Larger example with predication that should also not be vectorized
-; CHECK-LABEL: predicated
+; CHECK-LABEL: predicated_test
```

Comment:
Without this change, the test was failing due to name cluttering

Reply:
Can you adjust the name in a separate commit?